packages/cli/src/Studio.ts

```diff
@@ -106,7 +94,7 @@ ${bold('Examples')}
     const staticAssetDir = path.resolve(__dirname, '../build/public')
 
     const schema = await fs.promises.readFile(schemaPath, 'utf-8')
-    const config = await getConfig({ datamodel: schema, ignoreEnvVarErrors: false })
+    const config = await getConfig({ datamodel: schema, ignoreEnvVarErrors: true })
```

Comment:
Not required to throw an error here for env vars, Studio will display one anyways.

Reply:
Why changing it if "it's the same"?

Reply:
This was changed recently to make `directUrl` work but is not actually necessary because Studio will present an error on its own and in the UI, this happens via the CLI. Additionally, the reason why I needed to change this is because the engine also validates the Data Proxy URLs and errors.